static/js/settings_org.js

```diff
@@ -481,37 +481,36 @@ function _set_up() {
         }
     });
 
-    $('.organization').on('click', '.subsection-header .subsection-changes-discard button', function (e) {
-        e.preventDefault();
-        e.stopPropagation();
-
-        var properties_elements = get_subsection_property_elements(this);
+    function discard_subsection_changes(target) {
+        var properties_elements = get_subsection_property_elements(target);
         _.each(properties_elements, function (elem) {
             elem = $(elem);
             var property_name = exports.extract_property_name(elem);
-            if (typeof page_params[property_name] === 'boolean') {
-                // We trigger a click event rather than just changing
-                // the prop, because that handles dependent
-                // sub-settings correctly.
-                if (elem.prop('checked') !== page_params[property_name]) {
-                    elem.click();
-                }
-            } else if (typeof page_params[property_name] === 'string' || typeof page_params[property_name] === 'number') {
-                elem.val(page_params[property_name]);
-            } else {
-                // Check whether the id refers to a property whose name we can't
-                // extract from element's id.
-                var property_value = property_value_element_refers(property_name);
+            // Check whether the id refers to a property whose name we can't
+            // extract from element's id.
+            var property_value = property_value_element_refers(property_name);
```

Comment:
This `var` is redundant.

Reply:
How is this `var` redundant? It is used multiple times and this contains the value that property element should have in dom(below `if` statement puts the page_params value if it doesn't need to be treated specially.

Reply:
Oops, I just misread this as being `var property_name` again, never mind :)